typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/helpers/findTargetItem.ts

```diff
@@ -6,35 +6,53 @@
  * %%
  * Build Tools for VMware Aria
  * Copyright 2023 VMware, Inc.
- * 
+ *
  * This product is licensed to you under the BSD-2 license (the "License"). You may not use this product except in compliance with the BSD-2 License.
- * 
+ *
  * This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
  * #L%
  */
 import { WorkflowItemDescriptor } from "../../../../decorators";
 import { findItemByName } from "./findItemByName";
 
+export const DEFAULT_END_ITEM_NAME = "item0";
```

Comment:
maybe you can move this to another file altogether with constants?

Reply:
Will add helper presentationConstants.ts with default end item name, start item name, start item offset (as there is no strategy to cover for it)